config/locales/en.yml

```diff
@@ -373,6 +374,9 @@ en:
       root: *root
     result:
       root: *root
+    admin:
+      internal_categories:
+        root: "Internal Categories"
```

Comment:
i18n should be added to all language files as comments to mark untranslated keys